src/views/Home.jsx

```diff
@@ -31,37 +32,53 @@ export function Home({ data, setListPath, userId, userEmail }) {
 	const handleChange = (e) => {
 		setName(e.target.value);
 	};
+	// to create a highlighted list
+	const handleListClick = (path) => {
+		setListPath(path);
+		setSelectedList(path); // this update and highlight the list path
+	};
 
 	return (
-		<div className="Home">
-			<p>
-				Hello from the home (<code>/</code>) page!
-			</p>
-
+		<div className="home">
 			<form id="list" onSubmit={handleSubmit}>
-				<label htmlFor="listName">Name of shopping list: </label>
+				<label htmlFor="listName" className="pt-8 font-bold text-center">
+					Kindly generate a shopping Collection
+				</label>
 				<br />
 				<input
 					type="text"
 					id="listName"
 					value={name}
 					onChange={handleChange}
+					className="input"
+					placeholder="Type Here..."
 					required
 				/>
 				<br />
-				<button type="submit">Create list</button>
+				<div className="btn">
+					<button type="submit">Register Collection List </button>
+				</div>
 			</form>
-
 			<ul>
 				{data?.map((item) => (
 					<SingleList
 						key={item.path}
 						name={item.name}
 						path={item.path}
-						setListPath={setListPath}
+						setListPath={handleListClick}
+						deleteList={deleteList} //  the deleteList function is being passed as a prop
+						selected={selectedList === item.path} // Checks if the list is selected
```

Comment:
as I said above instead of the `selectedList`, use the `lisrPath` from the parent.

Reply:
okay

Reply:
Fixed